### ISSUE TITLE
Webpacking Browserify bundles

### ISSUE THREAD

**Issue by guybedford** [zeit/ncc#91]
For the bug in https://github.com/zeit/ncc/pull/87, the problem seems to be that `require('error-stack-parser')` is an external require, but this cannot be statically analyzed from the ~~Webpack~~ Browserify CommonJS bundle because the external requires and internal requires in a  ~~Webpack~~ Browserify bundle all look the same to Webpack.

That is, Webpacking ~~Webpack~~ Browserify output doesn't work as we can't statically analyze the external requires. ~~Ideally Webpack would have this "transitivity" property built into its output or input handling... but it doesn't unfortunately.~~

What can likely be done here, is very careful static detections of ~~Webpack~~ Browserify bundles themselves, with an ability to thereby determine the externals.

For example, in the bugsnag output, the hint that `error-stack-parser` is an external seems to come from the contextual object associated with the module that has that require:

```js
},{"./lib/es-utils":7,"./lib/has-stack":8,"error-stack-parser":undefined,"stack-generator":undefined}],21:[function(require,module,exports){
```

By reading the "~~webpack~~ browserify output structure", and determining these contextual objects, it should then be possible to enable this transitive Webpacking.

Ideally such a feature would be implemented in Webpack core itself though. I'm sure @sokra already has a solution / thoughts on this as well.

**Comment by guybedford** [comment]
//cc @sokra

**Comment by guybedford** [comment]
Thanks @sokra for the correction here! Of course you handle this already :)

The same arguments as above apply here to Browserify bundles then effectively, and still needs to be solved here.

**Comment by guybedford** [comment]
Fixed in #87.
